### PR TITLE
feat: add CesiumPy Viewer to Python utilities

### PR DESCRIPTION
**Pull request by lucas-bremond**
This PR adds a 3D viewer based on [CesiumPy](https://github.com/lucas-bremond/cesiumpy) to the Python utilities, to simplify displaying Profiles in Python.

image

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -471,6 +471,7 @@ test-unit-python-standalone: ## Run Python unit tests (standalone)
 		/bin/bash -c "cmake -DBUILD_PYTHON_BINDINGS=ON -DBUILD_UNIT_TESTS=OFF .. \
 		&& $(MAKE) -j 4 && python3.11 -m pip install --root-user-action=ignore bindings/python/dist/*311*.whl \
 		&& python3.11 -m pip install plotly pandas \
+		&& python3.11 -m pip install git+https://github.com/lucas-bremond/cesiumpy.git#egg=cesiumpy \
```
Comment (vishwaLoft) [inline Makefile:474]: Can do in a separate MR but should we just make a `test-requirements.in`?

File: bindings/python/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Apache License 2.0
 
-open-space-toolkit-physics~=1.1.1
+open-space-toolkit-physics~=1.1
```
Comment (vishwaLoft) [inline bindings/python/requirements.txt:3]: Is there a reason you're pinning to the major version? I think we had decided that we will pin to the minor version no?

**Review by vishwaLoft — APPROVED** [review]
This is freaking sick! Just some nits.

File: bindings/python/requirements.txt (hunk 2 above)
Comment (kyle-cochran) [reply bindings/python/requirements.txt:3]: +1 I believe this will allow Python to automatically update minor versions, which is different than how cpp dependencies get propagated. This caused some syncing issues between cpp/Python in the past.

**Review by kyle-cochran — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (lucas-bremond) [reply Makefile:474]: Yeah that'd be great!

File: bindings/python/requirements.txt (hunk 2 above)
Comment (lucas-bremond) [reply bindings/python/requirements.txt]: Reverted!
